.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/patternfly-core"]
+	path = submodules/patternfly-core
+	url = https://github.com/patternfly/patternfly.git
```

Comment:
One can simply checkout the tag in the submodule directoy, and commit the result in the parent.